packages/panels/src/Commands/MakeIssueCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Filament\Commands;
+
+use Illuminate\Console\Command;
+
+class MakeIssueCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'make:filament-issue';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generates a link to the Filament issue page and pre-fills the issue template.';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): void
+    {
+        $filament = \Composer\InstalledVersions::getPrettyVersion('filament/filament');
+        $laravel = \Composer\InstalledVersions::getPrettyVersion('laravel/framework');
+        $livewire = \Composer\InstalledVersions::getPrettyVersion('livewire/livewire');
+        $php = \PHP_VERSION;
+
+        $issueTemplate = 'bug_report.yml';
+        $baseGithubUrl = 'https://github.com/filamentphp/filament/issues/new';
+
+        $queryParams = [
+            'template' => $issueTemplate,
+            'package-version' => $filament,
+            'laravel-version' => $laravel,
+            'livewire-version' => $livewire,
+            'php-version' => $php,
+        ];
+
+        $queryString = http_build_query($queryParams);
+
+        $fullUrl = $baseGithubUrl . '?' . $queryString;
+
+        $this->info('Please fill out the issue template at ' . $fullUrl);
```

Comment:
You can actually redirect here instead, check out `askToStar()` in `InstallCommand`